package.json

```diff
@@ -451,7 +452,7 @@
         "id": "vscode-objectscript-output",
         "aliases": [],
         "mimetypes": [
```

Comment:
I haven't tried this, but I think it should be OK to remove the `mimetypes` array property completely now.